nav2_bringup/bringup/params/nav2_params.yaml

```diff
@@ -68,6 +68,8 @@ bt_navigator:
     - nav2_rate_controller_bt_node
     - nav2_distance_controller_bt_node
     - nav2_speed_controller_bt_node
+    - nav2_truncate_path_controller_bt_node
```

Comment:
Add these to the hardcoded list in BT navigator as well for defaults

Reply:
ok